### ISSUE TITLE
fail to install via poetry 

### ISSUE THREAD

**Issue by CircleOnCircles** [pydata/pandas-gbq#319]
```
(.venv) PS C:\Users\lifep\Devs\jobsbot> poetry add pandas-gbq
Using version ^0.13.2 for pandas-gbq

Updating dependencies
Resolving dependencies...

Writing lock file


Package operations: 16 installs, 0 updates, 0 removals

  - Installing pyasn1 (0.4.8)
  - Installing cachetools (4.1.0)
  - Installing protobuf (3.12.0)

[EnvCommandError]
Command ['C:\\Users\\lifep\\Devs\\jobsbot\\.venv\\Scripts\\pip.exe', 'install', '--no-deps', 'protobuf==3.12.0'] errored with the following return code 1, and output:
Collecting protobuf==3.12.0
  ERROR: Could not find a version that satisfies the requirement protobuf==3.12.0 (from versions: 2.0.0b0, 2.0.3, 2.3.0, 2.4.1, 2.5.0, 2.6.0, 2.6.1, 3.0.0a2, 3.0.0a3, 3.0.0b1, 3.0.0b1.post1, 3.0.0b1.post2, 3.0.0b2, 3.0.0b2.post1, 3.0.0b2.post2, 3.0.0b3, 3.0.0b4, 3.0.0, 3.1.0, 3.1.0.post1, 3.2.0rc1, 3.2.0rc1.post1, 3.2.0rc2, 3.2.0, 3.3.0, 3.4.0, 3.5.0.post1, 3.5.1, 3.5.2, 3.5.2.post1, 3.6.0, 3.6.1, 3.7.0rc2, 3.7.0rc3, 3.7.0, 3.7.1, 3.8.0rc1, 3.8.0, 3.9.0rc1, 3.9.0, 3.9.1, 3.9.2, 3.10.0rc1, 3.10.0, 3.11.0rc1, 3.11.0rc2, 3.11.0, 3.11.1, 3.11.2, 3.11.3)
ERROR: No matching distribution found for protobuf==3.12.0
```

There is a deps conflict. Even `pandas-gbq` is the only package in the project.

## To reporduce
1. poetry init
2. poetry add pandas-gbq

**Comment by max-sixty** [comment]
I don't think pandas-gbq requires protobuf directly (https://github.com/pydata/pandas-gbq/blob/master/setup.py)

So I'm guessing this is an issue with one of pandas-gbq's dependencies?

**Comment by tswast** [comment]
Both `google-cloud-bigquery` and `google-cloud-bigquery-storage` require protobuf.

* `google-cloud-bigquery` because of BQML resources
* `google-cloud-bigquery-storage` because of GRPC

**Comment by CircleOnCircles** [comment]
`python-bigquery-storage` -> `python-api-core` -> `"protobuf >= 3.4.0"`

Not sure which on require `protobuf==3.12.0`

**Comment by tswast** [comment]
We recently added a direct dependency on `protobuf>=3.12.0` in the `google-cloud-bigquery` package. https://github.com/googleapis/python-bigquery/pull/306 Even though this is redundant with some `google-api-core` dependencies, it does allow us to test explicitly the minimum version over there.

We had a similar problem with the pyarrow dependency recently. https://github.com/pydata/pandas-gbq/issues/324 I marked that as fixed by updating the dependency on `google-cloud-bigquery` to include the necessary "extras" from that package.

@CircleOnCircles  As we've made several changes to dependencies since this was opened, I'm going to mark this as closed, but please do re-open / let us know if it is not fixed in 0.14.0
